playbooks/roles/tower/defaults/main.yml

```diff
@@ -193,3 +190,9 @@ tower_email_notification_template_recipients: '{{ send_grid_email_recipients }}'
 tower_email_notification_template_host: '{{ send_grid_host }}'
 tower_email_notification_template_port: '{{ send_grid_port }}'
 
+tower_configuration_project_scm_branch: '{{ integreatly_version | default("master") }}'
+tower_credentials_project_scm_branch: '{{ integreatly_version | default("master") }}'
```

Comment:
Is there a reason why we moved these vars down to the bottom of the file?

Reply:
No real reason, I'll move them back up